frontend/src/components/GroupDetailsPage/EventDetails.jsx

```diff
@@ -0,0 +1,34 @@
+const EventDetails = ({ event: { name, previewImage, startDate }, group: { city, state } }) => {
+    const date = new Date(startDate);
+    const day = date.getDay();
+    const month = date.getMonth();
+    const year = date.getFullYear();
+    let time = date.toLocaleTimeString();
+    const amOrPm = time.slice(time.indexOf('M') - 1);
+    time = time.split(':').splice(0, 2).join(':');
+
+
+    return (
+        <>
+            <div className="group-details-event-top">
+                <img src={previewImage} className="group-details-event-img" />
+                <div className="group-details-event-header">
+                    <span className="group-details-event-span">
+                        <p className="group-details-event-date">{year}-{month}-{day}</p>
+                        <p className="group-details-event-date centered-dot">.</p>
+                        <p className="group-details-event-date">{time} {amOrPm}</p>
+                    </span>
+                    <h2 className="group-details-event-h2">{name}</h2>
+                    <p className="group-details-event-p">{city}, {state}</p>
+                </div>
+            </div>
+            <div className="group-details-event-bottom">
+                <p className="group-details-event-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque, perferendis! Nam fugit aliquam voluptatum eius aut nobis laudantium iste. Quam natus accusantium, recusandae deleniti voluptates rem quis assumenda! Voluptatibus, est.
```

Comment:
Are you going to render the same text for each event? Or do you plan to render the data from event.descriptions eventually?

Reply:
I might have missed that I have descriptions to display lol.

Reply:
Oh I forgot that getting the Event by Id gets some more information.